frontend/pages/queries/ManageQueriesPage/components/QueriesTable/QueriesTableConfig.tsx

```diff
@@ -198,6 +210,38 @@ const generateTableHeaders = ({
         />
       ),
     },
+    {
+      title: "Automations",
+      Header: "Automations",
+      disableSortBy: true,
+      accessor: "automations_enabled",
+      Cell: (cellProps: IStringCellProps): JSX.Element => {
+        let status;
+        if (cellProps.cell.value) {
+          if (cellProps.row.original.interval === 0) {
+            status = "paused";
+          } else {
+            status = "on";
+          }
+        } else {
+          status = "off";
+        }
+
+        const tooltip =
+          status === "paused"
+            ? {
+                id: cellProps.row.original.id,
+                tooltipText: (
+                  <>
+                    <strong>Automations</strong> will resume for this query when
+                    a frequency is set.
+                  </>
+                ),
+              }
+            : undefined;
+        return <StatusIndicator value={status} tooltip={tooltip} />;
+      },
+    },
```

Comment:
Since the rendering is not straightforward here, I would recommend you put the rendering logic in a separate component and have the complexity live there instead of in this table config file.